ament_cmake_auto/cmake/ament_auto_package.cmake

```diff
@@ -25,6 +25,9 @@
 #   Libraries are not affected by this option.
 #   They are always installed into `lib` and `dll`s into `bin`.
 # :type INSTALL_TO_PATH: option
+# :param INSTALL_TO_SHARE: a list of folders in the current folder that
+#   should be installed to the package's share folder
+# :type INSTALL_TO_SHARE: list of folder names
```

Comment:
Please change this to:

```
# :param INSTALL_TO_SHARE: a list of directories to be installed to the
#   package's share folder
# :type INSTALL_TO_SHARE: list of strings
```

It should also work with nested relative paths like `foo/bar` as well as absolute paths.